spec/features/courses/subjects/new_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
```

Comment:
I wonder about the value of this test ... it basically tests a subset of the "new course" feature, but because it uses Capybara it's going to be slow. If we want additional unit-level tests, would it make sense to rely on request specs instead?

Reply:
I'm not sure how I could do any useful testing of this functionality with request specs.

Reply:
Discussed IRL, decided there's value in testing this page in isolation as a feature test, as errors from previous steps could change the behaviour of this one and lead to false positive/negatives. 👍